reana_client/cli/workflow.py

```diff
@@ -146,22 +146,22 @@ def workflow_create(ctx, file, user, organization, skip_validation):
 
     click.echo('Workflow `{}` has been created.'.format(get_random_name()))
```

Comment:
Should this be changed and moved to end (of this function) or commented out for now?